src/encoded/static/components/immune_cells.js

```diff
@@ -1039,6 +1039,8 @@ class ImmuneCells extends React.Component {
                                         }
                                         key={this.state.availableNodes.length}
                                     />
+                                : (this.state.facetsOpen) ?
+                                    <MatrixGraph context={this.state.pageContext} setFilters={this.setNodeFilters} availableNodes={this.state.availableNodes} key={this.state.nodes.length} selectedNodes={this.state.nodes} />
```

Comment:
Nice! Thanks for doing that. This can run afoul of the many who hate nested conditional expressions, but I’ve never had an issue with those as long as it’s clear, and you wrote it clearly here.